src/seedu/addressbook/parser/Parser.java

```diff
@@ -54,7 +54,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```

Comment:
based on the files changed, I am guessing, you have not edited that part of the code where you create a parser object.
after making this method static, you can remove the parser object from your code!